### PR TITLE
Add KR locale

### LINKED ISSUE #253: Add South Korea language
Please add KR language :heart:

### PR DESCRIPTION
**Pull request by dimitriBouteille**
Add :kr:  locale

fixed #253

### CODE REVIEW

File: src/localization.ts [hunk 1]
```diff
@@ -750,6 +750,26 @@ const LANG_TH: Localization = {
   text_fetch_error: "ไม่สามารถเชื่อมต่อได้"
 };
 
+// South Korean
+const LANG_KR: Localization = {
+  text_init: "초기화 중",
+
+  text_ready: "Anti-Robot 검증",
```
Comment (merlinfuchs) [inline src/localization.ts:757]: I'm guessing there is no good translation for `Anti-Robot` in Korean?

**Review by merlinfuchs — APPROVED** [review]
Thanks a lot! 🚀 

Could you also add the new language to the list of supported languages in `docs/widget_api.md` and to the changelog in `docs/changelog.md`? Otherwise, I will do it when the PR is merged :)

File: src/localization.ts (hunk 1 above)
Comment (dimitriBouteille) [reply src/localization.ts:757]: Very good question, for a personal need, I had translated by a person who speaks Korean, the translation is good I think :see_no_evil:

**Comment by gzuidhof** [comment]
Thank you @dimitriBouteille!

**Pull request merged**